store/boltdb/boltdb_test.go

```diff
@@ -0,0 +1,54 @@
+package boltdb
+
+import (
+	"testing"
+
+	"github.com/docker/libkv/store"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBoltDBStore(t *testing.T) {
```

Comment:
Would be better if we can use the existing tests in `testutils/utils.go`/

Probably by decomposing the `TestStore` function in `testutils/utils.go` into a common subset `TestCommon` and then `TestLock`, `TestWatch`, etc. that BoltDB would ignore.

WDYT?